whitepapers/economic-productivity.md

```diff
@@ -67,17 +67,17 @@ The policy levers are the main way that the DAO self-regulates irrational, runaw
+**Premium over RFV**: this is not a *policy* lever, but a market measure. It is the value that each $OHM token trades for above the amount of stablecoin value backing each token. This is a multiple, comparable to a price to earnings multiple that is familiar to value investors. The premium is the external / price measure of internal coordination; the reason that $OHM trades for a price over the RFV is because there is an external market perception of efficient internal coordination of OlympusDAO contributors. This external perception reflects investor confidence that the percentage of $OHM staked will remain high, that contributors will continue to work for the DAO, that the protocol will continue to expand its network to form new partnerships, and that demand for $OHM will continue to increase. Thus, the premium over RFV is thus a measure of the economic productivity of the DAO, and its expected future cashflows. This measure is set by the market, and not directly by the DAO policy team, but it can be influenced by policy levers.
 
 **4. How These Aspects Create an Economic Flywheel**
 
-This is an *idealized prototype* of the economic flywheel mechanism, meant to be pedagogical rather than precise in detail. It gives an intuition for how the protocol self-regulates and aligns incentives of the three main parties- market/bonders, stakers, DAO policy team. This model shows how the implementation generalizes the economic forces of supply and demand, in order to work with or counteract reflexivity in the market. The bolded terms are the main levers that the DAO can use to internally affect market conditions.
+This is an *idealized prototype* of the economic flywheel mechanism, meant to be pedagogical rather than precise in detail. It gives an intuition for how the protocol self-regulates and aligns incentives of the three main parties- market/bonders, stakers, DAO policy team. This model shows how the implementation generalizes the economic forces of supply and demand, in order to work with or counteract reflexivity in the market.
```

Comment:
```suggestion
This is an *idealized prototype* of the economic flywheel mechanism, meant to be pedagogical rather than precise in detail. It gives an intuition for how the protocol self-regulates and aligns incentives of the three main parties- market/bonders, stakers, and the DAO policy team. This model shows how the implementation generalizes the economic forces of supply and demand, in order to work with or counteract reflexivity in the market.
```